Create CODE_OF_CONDUCT.md (#100)
greenelab/covid19-review: README.md

[![Gitter](https://badges.gitter.im/covid19-review/community.svg)](https://gitter.im/covid19-review/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
<!-- usage note: delete CI badges above for services not used by your manuscript -->

## Code of Conduct

> Participation in this project requires that you adhere to the [code of conduct](CODE_OF_CONDUCT.md).

Reviewer: This could be a place to indicate how contributors are expected to sign the code of conduct.

Author: If @agitter and @cgreene think 038d116 looks good, we should be OK to merge

> Participation in this project requires that you adhere to the [code of conduct](CODE_OF_CONDUCT.md).

Reviewer 2: ```suggestion
This project operates under a code of conduct.
Participating in the project in any way (issues, pull requests, gitter, or other media) indicates that you agree that you will follow the [code of conduct](CODE_OF_CONDUCT.md).
```

> We take this very seriously.

Reviewer 2: ```suggestion
Participating in the project in any way (issues, pull requests, gitter,  or other media) indicates that you agree that you will follow this code of conduct.
We take this very seriously.
```

Reviewer: Do you want to keep the link to the code of conduct here?  It may not be clear what "this code" is otherwise.

Reviewer 2: Agree - revised 👍

> If you have any issues with harassment or with anyone violating the code of conduct, please seek out one of the project organizers (@rando2 or @cgreene).

Reviewer 2: ```suggestion
If you experience harassment or notice violations of the code of conduct, please raise the issue to one of the project organizers (@rando2 or @cgreene).
```
